DEVELOPMENT.md

```diff
@@ -2,16 +2,18 @@ Make sure react-native-purchases is not part of the `example/package.json`
 
 Setup the development by running: 
 
-    `yarn bootstrap`
+```
+yarn bootstrap
```

Comment:
Formatting for this was off too:
<img width="199" alt="Screenshot 2023-01-20 at 15 41 06" src="https://user-images.githubusercontent.com/685609/213824202-d0683924-092c-4410-a2f9-b72da373290d.png">